.github/workflows/prepare-release.yml

```diff
@@ -85,6 +87,21 @@ jobs:
             ### Major Changes
 
             See [CHANGELOG.md](https://github.com/GoogleContainerTools/jib/blob/master/jib-${{ github.event.inputs.project }}-plugin/CHANGELOG.md) for more details.
+      
+      - uses: JasonEtco/create-an-issue@v2
+        if: ${{ github.event.inputs.project == 'maven' || github.event.inputs.project == 'gradle' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PROJECT_NAME: jib-${{ github.event.inputs.project }}-plugin
```

Comment:
This seems unnecesary.

Reply:
Ah right, an artifact from before. I'll remove